notes/stories/literature/shortstory-amazingstories.st.txt

```diff
@@ -1221,3 +1221,40 @@ folk belief [Some of the workmen thought that an Incan devil or evil spirit had
 exponential growth [Barry observed that the murders were approximately doubling by the day, which led Henshaw to conclude that everyone would be wiped out within a week.]
 
 
+shortstory: The Man who Saw the Future (1930)
+=============================================
+
+:: Title
+The Man who Saw the Future
+
+:: Date
+1929-12
+
+:: Description
+A 15th century Frenchman is plucked out of his own time and given a tour of
+1944 Paris.
+
+:: Authors
+Edmond Hamilton
+
+:: References
+https://www.gutenberg.org/ebooks/28062
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+what if I found myself in the future [The story follows the apothecary's assistant Henri Lothiere as he is transported by a pair of scientists from 1444 Paris to the Paris of 1944.]
+
+:: Major Themes
+time travel [The two scientists Rastin and Thicourt used some or another machine they'd constructed to transport Henri Lothiere from 1444 Paris to their own time and place of 1944 Paris.]
+what if I told the truth and nobody would believe me [Nobody in Henri Lothiere's own time of 1444 believed his story that he'd traveled to 1944 Paris and see numerous technological marvels. Instead, he was labeled a sorcerer and a blasphemer and ultimately burned at the stake.]
```

Comment:
Brings also to mind
https://www.themeontology.org/theme.php?name=science%20as%20magic%20to%20the%20primitive

Reply:
I deliberately left this out at first, but I found some content from one paragraph to justify its application. Added accordingly!